test/p2p-test.js

```diff
@@ -73,58 +72,6 @@ describe('P2P', function () {
     await node2.close();
   });
 
-  describe('BIP157', function () {
```

Comment:
Interesting that this change triggered a bunch of code coverage warnings, will have to investigate further.